### PR TITLE
Added Agent component

### LINKED ISSUE #1: Add Webpack build and config
Adding a webpack config and build to bundle and minify assets would reduce the "confusion" of the frontend and help to organize and structure project folder.

See: [webpack](https://webpack.js.org/)

### PR DESCRIPTION
**Pull request by Mardii-D-King**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | no
| Major: Breaking Change?  | no
| Minor: New Feature?      | yes
| Deprecations?            | no
| Spec Compliancy?         | no
| Tests Added/Pass?        | no
| Fixed Tickets            |n/a
| License                  | MIT
| Doc PR                   | no
| Dependency Changes       | no

The Agents component that will be used to show which agents are collecting data have been added to the src folder. The route for the link is also implemented.

### CODE REVIEW

File: routes/routes.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
-const express = require("express");
-const router = express.Router();
+const express = require ('express');
+const router = express.Router ();
 
 //Routes
+<<<<<<< HEAD
```
Comment (Pr0x1m4) [inline routes/routes.js]: Please pay close attention to files you commit, this file has a merge conflict. Please resolve

File: src/Agent/Agent.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
+/**
```
Comment (Pr0x1m4) [inline src/Agent/Agent.js]: The File contents say `Settings` but the file name is `Agent`. Was this intentional? Did you mean to right an Agent class?

File: src/App.js [hunk 3]
```diff
@@ -4,10 +4,17 @@
  * Properties: None
  **/
 
+<<<<<<< HEAD
```
Comment (Pr0x1m4) [inline src/App.js]: Another merge conflict present in this file. Please resolve.

File: src/Navbar/Navbar.js [hunk 4]
```diff
@@ -30,11 +30,15 @@ class NavBar extends Component {
           </li>
           <li>
             <Link to="/">
+<<<<<<< HEAD
```
Comment (Pr0x1m4) [inline src/Navbar/Navbar.js]: Another merge conflict. Please resolve

File: routes/routes.js (hunk 1 above)
Comment (Mardii-D-King) [inline routes/routes.js]: Doing it now

**Pull request merged**